lua/pl/text.lua

```diff
@@ -4,320 +4,23 @@
 -- libraries, see string.Template). It also provides similar functions to those
 -- found in the textwrap module.
 --
+-- IMPORTANT: this module has been deprecated and will be removed in a future
+-- version (2.0). The contents of this module have moved to the `pl.stringx`
+-- module.
+--
 -- See  @{03-strings.md.String_Templates|the Guide}.
 --
--- Dependencies: `pl.utils`, `pl.types`
+-- Dependencies: `pl.stringx`
```

Comment:
Removing the pl.utils dependency seems to be a mistake since the deprecation notice requires it.

Reply:
fixed